File: VB/WPFDataGridGettingStartedNETCore/Models/Region.vb
Imports System.Collections.Generic

Namespace WPFDataGridGettingStartedNETCore.Models

    Public Partial Class Region

        Public Sub New()
            Territories = New HashSet(Of Territory)()
        End Sub

        Public Property RegionId As Integer

        Public Property RegionDescription As String

        Public Overridable Property Territories As ICollection(Of Territory)
    End Class
End Namespace

File: CS/WPFDataGridGettingStartedDotNET/Models/Region.cs
using System;
using System.Collections.Generic;

namespace WPFDataGridGettingStartedDotNET.Models;

public partial class Region
{
    public int RegionId { get; set; }

    public string RegionDescription { get; set; }

    public virtual ICollection<Territory> Territories { get; } = new List<Territory>();
}
